packages/utils/models.js

```diff
@@ -164,178 +176,208 @@ export const ARROW_NAVIGATION_DEVICES = [
+ * @description If current model have only touch screen (no physical keyboard)
+ * @returns {boolean}
  */
 export function hasOnlyTouch() {
-  return _hasModelAtList(ONLY_TOUCH);
+  return !VerifyMethodOnSystemWrapper('hasKeyboard') || _hasModelAtList(ONLY_TOUCH);
 }
 
 /**
- * Devices with no touch capability
+ * @description Devices with no touch capability
  * @returns {array} A list of devices that doesn't have touch screen
  */
 export const NO_TOUCH = [MODELS.D195, MODELS.Q60, MODELS.D230];
 
 /**
- * If device doesn't have touch.
- * @param {boolean} useCache If should use stored value or not, to avoid
- * access bridge unnecessary, usually for defualt values across components.
- * @returns {boolean} If current model have no touch screen
+ * @description If current model have no touch screen
+ * @returns {boolean}
  */
-export function hasNoTouch(useCache = true) {
-  if (useCache === true && '_hasNoTouch' in ThisStore) {
-    return ThisStore._hasNoTouch;
-  }
-
-  const _hasNoTouch = _hasModelAtList(NO_TOUCH);
-  ThisStore._hasNoTouch = _hasNoTouch;
-  return _hasNoTouch;
+export function hasNoTouch() {
+  return !VerifyMethodOnSystemWrapper('hasTouch') || _hasModelAtList(NO_TOUCH);
 }
 
 /**
- * Devices with no touch capability
+ * @description Devices with no touch capability
  * @returns {array} A list of devices that have touch screen
  */
 export const WITH_TOUCH = [
   MODELS.S920,
   MODELS.Q92,
+  MODELS.Q92S,
   MODELS.MP35P,
   MODELS.MP35,
   MODELS.V240M,
   MODELS.D199,
 ];
 
 /**
- * @returns {boolean} If current model has touch screen
+ * @description If current model has touch screen
+ * @returns {boolean}
  */
 export function hasTouch() {
-  return _hasModelAtList(WITH_TOUCH);
+  return VerifyMethodOnSystemWrapper('hasTouch') || _hasModelAtList(WITH_TOUCH);
 }
 
 /**
- * Devices with no printer
+ * @description Devices with no printer
  * @returns {array} A list of devices that doesn't have printer
  */
 export const NO_PRINTER = [MODELS.MP35, MODELS.D199, MODELS.D195];
 
 /**
- * @returns {boolean} If current model have no printer
+ * @description If current model have printer
+ * @returns {boolean}
+ */
+export function hasPrinter() {
+  return VerifyMethodOnSystemWrapper('hasPrinter') || !_hasModelAtList(NO_PRINTER);
+}
+
+/**
+ * @description If current model have no printer
+ * @returns {boolean}
  */
 export function hasNoPrinter() {
-  return _hasModelAtList(NO_PRINTER);
+  return !VerifyMethodOnSystemWrapper('hasPrinter') || _hasModelAtList(NO_PRINTER);
 }
 
 /**
- * Devices with keyboard light
- * @returns {array} A list of devices that have keyboard light
+ * #####################
+ * ###### NETWORK ######
+ * #####################
  */
-export const HAS_KEYBOARD_LIGHT = [MODELS.MP35P, MODELS.D230, ...STANDARD_MAMBA_DEVICES];
 
 /**
- * @returns {boolean} If current model have keyboard light
+ * @description If POS has an Ethernet adapter
```

Comment:
(Detalhe) O que o método faz não precisa de `@description` no começo do comentário.
```suggestion
 * If POS has an Ethernet adapter
```
`@description` é pra você poder colocar a descrição em qualquer lugar nos comentários, (ex: depis dos parâmetros).

https://jsdoc.app/tags-description.html

Reply:
Pode resolver, se quiser. Isso aqui é totalmente opcional.

Reply:
entendi, a modificação valia pq a descrição tava junto do return. mas sim a tag em si de description é opcional.